apps/builder/app/builder/features/seo/image-control.tsx

```diff
@@ -0,0 +1,30 @@
+import { FloatingPanel } from "~/builder/shared/floating-panel";
+import { ImageManager } from "~/builder/shared/image-manager";
+import type { ReactElement } from "react";
+import { theme } from "@webstudio-is/design-system";
+
+export const ImageControl = (props: {
+  assetId: string;
+  onAssetIdChange: (assetId: string) => void;
+  children: ReactElement;
+}) => {
+  return (
+    <FloatingPanel
+      title="Images"
+      contentCss={{
+        // Left Aside panels (e.g., Pages, Components) use zIndex: theme.zIndices[1].
+        // For a panel to appear above these panels, both overlay and content should also have zIndex: theme.zIndices[1].
+        zIndex: theme.zIndices[1],
```

Comment:
if this is the only thing we need to pass over to content, might just pass zIndex or something